src/config/theme.js

```diff
@@ -417,50 +420,112 @@ bodhiTheme.overrides = {
       color: white,
       boxShadow: 'none',
       '&:hover': {
-        backgroundColor: primaryBrandColor80,
+        backgroundColor: primaryBrandColorTransparent80,
```

Comment:
can we just use the `opacity: 0.8` field here?

Reply:
In some places, the transparency of backgroundColor changes, but the transparency of text and border doesn't change.
![image](https://user-images.githubusercontent.com/18548885/48460389-c444e800-e809-11e8-9451-15de2640e6b2.png)